src/app/odasstream/odas_stream.py

```diff
@@ -29,21 +29,18 @@ def start(self):
 
 
     def stop(self):
-        if self.odasProcess and self.isRunning:
-            print('Stopping Odas stream...')
-            self.odasProcess.kill()
-            print('Odas stream stopped.')
-            self.isRunning = False
-
-            if self.odasProcess.returncode and self.odasProcess.returncode != 0:
-                raise Exception('ODAS exited with exit code {exitCode}'.format(exitCode=self.odasProcess.returncode))
+        self.isRunning = False
 
 
     def run(self):
         self.__spawnSubProcess()
 
+        print("ODAS stream started")
```

Comment:
When I ran the code, if I start ODAS without being connected to the microphone matrice it says the stream started even if some error was probably triggered. Pressing the stop button won't do anything and clicking the start button in repetition always says ODAS started.